src/screens.web.js

```diff
@@ -1,16 +1,38 @@
+import React from 'react';
 import { Animated, View } from 'react-native';
 
+let _shouldUseScreens = true;
+
 export function useScreens(shouldUseScreens = true) {
   if (shouldUseScreens) {
-    console.warn('react-native-screens is not support on this platform.');
+    console.warn('react-native-screens is not fully support on this platform yet.');
   }
+  _shouldUseScreens = shouldUseScreens;
 }
 
 export function screensEnabled() {
-  return false;
+  return _shouldUseScreens;
 }
 
-export const Screen = Animated.View;
+export class Screen extends React.Component {
+  render() {
+    const { style, active, ...rest } = this.props;
+    let viewStyle = [style];
+    let activeValue = active;
+    // Convert animated value to boolean.
+    if (active && active.__getValue) {
```

Comment:
Looks like this relies on rerender happening when animated value changes. Is that what you wanted here? Maybe we should set animated listener and update `display` style in it instead?